### ISSUE TITLE
Continious CI is required

### ISSUE THREAD

**Issue by xnox** [ibm-s390-tools/s390-tools#11]
Continious CI is required to test booting

- rootfs on LVM on DASD
- rootfs on multipath
- rootfs on zfcp / scsi
- rootfs on qemu-kvm virtio scsi

It seems like -f flag got dropped, without fixing zipl helper. Causing inability to boot since at at least v1.39.

**Comment by michael-holzheu** [comment]
I agree that CI tests would be definitely nice to have for *all* s390-tools. But it is a major effort with having all tools maintainers involved, choosing the right framework and somehow providing the required hardware infrastructure, e.g. DASDs or FCP attached SCSI devices.

At the moment I don't see how this is going to happen.

**Comment by markuszoeller** [comment]
@michael-holzheu 
The _LinuxOne Community Cloud_ is maybe able to offer the resources you're looking for: 
* https://developer.ibm.com/linuxone/
* https://linuxone20.cloud.marist.edu/cloud/#/login

I haven't yet used it personally though.
